src/Compilers/Core/MSBuildTask/Utilities.cs

```diff
@@ -152,7 +152,9 @@ internal static void DeleteNoThrow(string path)
 
         internal static string? TryGetAssemblyPath(Assembly assembly)
         {
+#pragma warning disable SYSLIB0005 // 'Assembly.GlobalAssemblyCache' is obsolete: 'The Global Assembly Cache is not supported.'
             if (assembly.GlobalAssemblyCache)
+#pragma warning restore SYSLIB0005
```

Comment:
Why is this check running against our code here now?

Reply:
This may have been an artifact of an earlier state. I will try to remove these.

Reply:
Note that this is just a case of a [normal ObsoleteAttribute](https://github.com/dotnet/runtime/blob/b8dbea7c618bf346e7d4a5f78ddb1f6e8abf67ea/src/libraries/System.Private.CoreLib/src/System/Reflection/Assembly.cs#L168) using SYSLIB0005 as the custom error code.